service/src/main/java/bio/terra/cbas/runsets/inputs/InputGenerator.java

```diff
@@ -42,9 +42,10 @@ public static Map<String, Object> buildInputs(
         parameterValue = null;
       } else if (param.getSource() instanceof ParameterDefinitionRecordLookup recordLookup) {
         String attributeName = recordLookup.getRecordAttribute();
-        parameterValue = recordResponse.getAttributes().get(attributeName);
 
-        if (!((Map<String, Object>) recordResponse.getAttributes()).containsKey(attributeName)) {
+        if (((Map<String, Object>) recordResponse.getAttributes()).containsKey(attributeName)) {
```

Comment:
does this change impact the main issue? or is it an add-on refactor?

(no complaints, I just don't immediately see how it relates to the autoboxing)

Reply:
Just noise/tidying, apologies!